examples/profile.js

```diff
@@ -0,0 +1,52 @@
+goog.provide('profile_example');
+
+
+goog.require('ngeo');
+goog.require('ngeo.profileDirective');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['ngeo']);
+
+
+
+/**
+ * @constructor
+ * @param {angular.$http} $http The $http angular service.
+ * @param {angular.Scope} $scope The $scope angular service.
+ */
+app.MainController = function($http, $scope) {
+
+  $http.get('data/profile.json').then(angular.bind(this, function(resp) {
+    this['profileData'] = resp.data;
+  }));
+
+  this['point'] = null;
+
+  this.scope_ = $scope;
+};
+
+
+/**
+ * @param {Object} point The point object with x, y, elevation, distance
+ * information.
```

Comment:
The object sent to callbacks is supposed to be the point as it was given to the profile. It's the raw data.